apps/wing-console/console/app/src/index.ts

```diff
@@ -69,10 +71,23 @@ export const createConsoleApp = async (options: CreateConsoleAppOptions) => {
         0,
         Math.max(0, trace.data.message.indexOf("(")),
       );
+
+      // general interaction event
+      analytics.track(
+        'console_resource_interact',
+          {
+            resource: resourceName,
+            action,
+          }
+      );
+      // resrouce specific event
       analytics.track(
-        `console application: ${resourceName}: ${action} ${JSON.stringify(
-          Object.assign({}, trace, trace.data),
-        )}`,
+        `console_${resourceName}_${action}`,
+          {
+            message: trace?.data?.message.substring(0, MAX_ANALYTICS_STRING_LENGTH) || '',
+            status: trace?.data?.status.substring(0, MAX_ANALYTICS_STRING_LENGTH) || 'unknown',
+            result: trace?.data?.result.substring(0, MAX_ANALYTICS_STRING_LENGTH) || 'unknown',
+          }
```

Comment:
lets add this data to the console_resource_interact as well

Reply:
ok